docs/src/how-to/change-username.md

```diff
@@ -0,0 +1,19 @@
+# How to change the username
+
+If you want to change the default username to something other than `nixos`, use the `wsl.defaultUser` option.
+When building your own tarball, this should be sufficient. A user with the name specified in that option will be created automatically.
+
+Changing the username on an already installed system is possible as well.
+Follow these instructions to make sure, the change gets applied correctly:
+
+1. Change the `wsl.defaultUser` setting in your configuration to the desired username.
+2. Apply the configuration:\
+   `nixos-rebuild boot`\
```

Comment:
```suggestion
   `sudo nixos-rebuild boot`\
```
I suggest either writing it like this or adding a note that you need to be root / sudo'ing for this. I edited my nixos config via `sudo nano /etc/nixos/configuration.nix` and then I got an error applying this command.

Reply:
Done, thanks for the input!